dev/attest/src/assertions/assertions.ts

```diff
@@ -77,12 +77,19 @@ export class Assertions implements AssertionRecord {
 
     // Use variadic args to distinguish undefined being passed explicitly from no args
     snap(...args: [expected: unknown]) {
+        const snapName = (args.at(1) ?? "snap") as string
         const expectedSerialized = this.serialize(args[0])
-        if (!args.length || this.ctx.cfg.updateSnapshots) {
+        if (
+            (args[0] === undefined && args.at(1) !== undefined) ||
+            this.ctx.cfg.updateSnapshots
+        ) {
             if (this.snapRequiresUpdate(expectedSerialized)) {
                 const snapshotArgs: SnapshotArgs = {
-                    position: caller(),
-                    serializedValue: this.serializedActual
+                    position: caller({
```

Comment:
Maybe we revert these now that they're not needed. People will be confused by the nonexistent hack 😅